app/helpers/application_helper.rb

```diff
@@ -73,4 +73,9 @@ def urgency_level_display(urgency_level_text)
       urgency_level_text
     end
   end
+
+  def site_logo_url
+    # There should always be a current org, but being defensive here helps simplify tests
+    Organization.instance_owner&.logo_url.presence || asset_pack_path('media/images/logo.png')
```

Comment:
Thank you! I've tripped up on this many times, and this feels to me like a much better home for this conditional than anywhere else it has lived before